src/components/Steam.vue

```diff
@@ -122,13 +124,23 @@ export default Vue.extend({
 <style scoped lang="scss">
 .title {
   max-height: 48px;
-  overflow: hidden;
   padding: 10px;
+  overflow: hidden;
+  overflow-x: scroll;
+  scrollbar-width: none;
+  -ms-overflow-style: none;
+  &::-webkit-scrollbar {
+    height: 0;
+  }
+  &-text {
+    white-space: nowrap;
+    padding-right: 10px;
+  }
```

Comment:
Is this duplicating the same functionality from the other component?
At what point is it worth it to add CSS directives into a global stylesheet?

Reply:
I wonder about this too. I do like that the component is "contained". I know it feels like repeating and it'll only get worse if we add an Epic, Origin, Gog component, etc. Perhaps there needs to be a "parent" component that they all extend? Is there a pattern for that in Vue? I imagine that CSS scope will be the main issue.